### DISCUSSION TITLE
NHibernate 5.4 Release

### DISCUSSION THREAD

**Discussion by NecatiMeral** [nhibernate/nhibernate-core#3186] [General] [answered]
Hey everyone

Is there a estimated release date for 5.4? The issues milestone says it's overdue since 18 days but that doesn't help.

Keep on pushing on!

**Comment by fredericDelaporte** [comment, marked as answer]
You may use the [MyGet feed](https://github.com/nhibernate/nhibernate-core#nightly-development-builds) instead if needing 5.4 as soon as possible.

**Comment by fredericDelaporte** [comment]
Now you can also look at #3195.
